tools/Ecoregionalization_workflow/BRT_model.xml

```diff
@@ -0,0 +1,110 @@
+<tool id="brt_analysis" name="BRT tool prediction" version="0.1.0+galaxy0" >
+    <description>for species distribution modelling</description>
+    <requirements>
+       <requirement type="package" version="4.3.0">r-base</requirement>
+       <requirement type="package" version="1.3_14">r-dismo</requirement>
+       <requirement type="package" version="2.1.8">r-gbm</requirement>
+       <requirement type="package" version="3.4.2">r-ggplot2</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript
+         '$__tool_directory__/brt.R'
+         '$enviro'
+         '$species_files'
+         '$input3'
+         '$output1'
+         '$output2'
+         '$output3'
+         '$output4'
+    ]]></command>
+    <inputs>
+      <param name="enviro" type="data" format="txt,csv,tsv" label="Environment Data"/>
+      <param type="data" name="species_files" label="Occurrences Data File" format="tabular,txt,csv" multiple="True" />
+      <param name="input3" type="text" label="write the name of your abiotic parameters (comma separated)"/>
+    </inputs>
+    <outputs>
+      <collection name="output1" type="list" label="Prediction files">
```

Comment:
Can you get multiple output files here? Under which circumstances can this happen?

Reply:
Yes, there is one output "Prediction file" per input "Occurrences Data File". So, when a user provides multiple input files then there will be multiple output files here.